test/test_images/sleeptalker/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+func main() {
```

Comment:
Thinking out loud here: I wonder if there's a way to make this test a little easier to reason about. What if the goroutine just printed "CONTAINER IS RUNNING" on each tick, and then print a single "container is stopped" (via a `defer`?) when the goroutine exits? Then to check if the container is running / paused just involves checking what the last log line says the state of the container is... what do you think?

Reply:
what u think, which way better?